.github/CODEOWNERS

```
@@ -9,6 +9,10 @@
 /website/src/pages/dbt-cloud/                          @runleonarun 
 /website/static/img/docs/dbt-cloud/                    @runleonarun 
 
+# Blog content assigned to Developer experience team
+/website/blog/                       @ganz @gwenwindflower @joellabes
```

```suggestion
/website/blog/                       @dbt-labs/devex 
```

Future-proof, [as long as the devex group has write access](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#:~:text=teams%20must%20have%20explicit%20write%20access)

---

Jinx, you wrote the same thing while I was looking this up 😅

---

Ha! Let's make this change and ask Jason to update the team!